src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/PackagingTest.cs

```diff
@@ -320,6 +325,7 @@ protected override void OnResume()
 				IsRelease = true,
 				UseLatestPlatformSdk = true,
 				References = {
+					new BuildItem.Reference ("Java.Interop.Export"),
```

Comment:
How'd we manage to not need this before? If we didn't need it before, why's it needed now?

Reply:
I think this is related to what @pjcollins saw in his tests. Not sure why I needed to add `Java.Interop.Export`, bit the test wouldn't build without it.. might have been a change upstream?